tempesta_fw/procfs.c

```diff
@@ -206,6 +208,24 @@ tfw_srvstats_seq_show(struct seq_file *seq, void *off)
 			atomic64_read(&srv->sess_n));
 	seq_printf(seq, "Total schedulable connections\t: %zd\n",
 			srv->conn_n - rc);
+
+	seq_printf(seq, "HTTP health monitor is enabled\t: %d\n", hm);
+	if (hm) {
```

Comment:
Perhaps, `hm_stats` should be declared within this block as it's not used elsewhere.

Reply:
Corrected.